.codeclimate.yml

```diff
@@ -26,6 +26,7 @@ engines:
       languages:
         ruby:
           mass_threshold: 25
+          count_threshold: 5
```

Comment:
What does tis control?

Reply:
It controls how many similar codes are allowed.